packages/rx_bloc_cli/mason_templates/bricks/feature_google_login/__brick__/lib/lib_auth_google/data_sources/remote/google_auth_data_source.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
```

Comment:
In all files inside the brick remove this, where applicable, and add to the top of files {{> licence.dart }} and empty row - see in other bricks.